File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, w1: str, w2: str) -> str:
        m = ""
        i = 0
        while i < min(len(w1),len(w2)):
            m+=w1[i]
            m+=w2[i]
            i+=1
        while i <len(w1):
            m+=w1[i]
            i+=1
        while i<len(w2):
            m+=w2[i]
            i+=1
        return m
        

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string s1, string s2) {
        int i =0,j=0;
        string s ;
        while(i<s1.size() and j<s2.size()){
            s+=s1[i];++i;
            s+=s2[j];++j;
        }
        while(i<s1.size()){
            s+=s1[i];++i;
        }
        while(j<s2.size()){
            s+=s2[j];++j;
        }
        return s;
    }
};